renderer/src/components/ChartsDashboard.jsx

```
@@ -140,107 +140,52 @@ export default function ChartsDashboard() {
-        <div className={`p-4 rounded-lg ${features.mapChartsEnabled ? 'bg-green-50 border border-green-200' : 'bg-gray-100 border border-gray-200'}`}>
+        <div className={`p-4 rounded-lg ${features.activityChartsEnabled ? 'bg-green-50 border border-green-200' : 'bg-gray-100 border border-gray-200'}`}>
           <div className="flex items-center">
-            <div className={`w-3 h-3 rounded-full mr-2 ${features.mapChartsEnabled ? 'bg-green-500' : 'bg-gray-400'}`}></div>
-            <span className="font-medium">Map Charts: {features.mapChartsEnabled ? 'Enabled' : 'Disabled'}</span>
+            <div className={`w-3 h-3 rounded-full mr-2 ${features.activityChartsEnabled ? 'bg-green-500' : 'bg-gray-400'}`}></div>
+            <span className="font-medium">Activity Charts: {features.activityChartsEnabled ? 'Enabled' : 'Disabled'}</span>
           </div>
         </div>
         <div className={`p-4 rounded-lg ${features.killChartsEnabled ? 'bg-green-50 border border-green-200' : 'bg-gray-100 border border-gray-200'}`}>
           <div className="flex items-center">
             <div className={`w-3 h-3 rounded-full mr-2 ${features.killChartsEnabled ? 'bg-green-500' : 'bg-gray-400'}`}></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)